docs/search/search_engines/elastic_search/configure_elastic_search.md

```diff
@@ -475,6 +475,180 @@ ibexa_elasticsearch:
+                            type: custom
+                            tokenizer: standard
+                            filter:
+                                - lowercase
+                                - english_possessive_stemmer
+                    filter:
+                        ibexa_spellcheck_shingle_filter:
+                            type: shingle
+                            min_shingle_size: 2
+                            max_shingle_size: 3
++                        english_stop:
++                            type: stop
++                            stopwords: "_english_"
++                        english_keywords:
++                            type: keyword_marker
++                            keywords: []
++                        english_stemmer:
++                            type: stemmer
++                            language: light_english
++                        "english_possessive_stemmer": {
++                          "type":       "stemmer",
++                          "language":   "possessive_english"
++                        }
+                refresh_interval: "-1"
+                index:
+                    mapping:
+                        total_fields:
+                            limit: 5000
+            mappings:
+                dynamic_templates:
+                    -   ez_int:
+                            match: "*_i"
+                            mapping:
+                                type: integer
+                    -   ez_mint:
+                            match: "*_mi"
+                            mapping:
+                                type: integer
+                    -   ez_id:
+                            match: "*_id"
+                            mapping:
+                                type: keyword
+                    -   ez_mid:
+                            match: "*_mid"
+                            mapping:
+                                type: keyword
+                    -   ez_string:
+                            match: "*_s"
+                            mapping:
+                                type: keyword
+                                normalizer: lowercase_normalizer
+                    -   ez_mstring:
+                            match: "*_ms"
+                            mapping:
+                                type: keyword
+                                normalizer: lowercase_normalizer
+                    -   ez_long:
+                            match: "*_l"
+                            mapping:
+                                type: long
+                    -   ez_mlong:
+                            match: "*_ml"
+                            mapping:
+                                type: long
++                    -   ez_text_fulltext:
++                            match: "*_fulltext"
++                            mapping:
++                                type: text
++                                analyzer: english_analyzer
```

Comment:
With [this PR](https://github.com/ezsystems/ezplatform-elastic-search-engine/pull/57) merged, the diff will look slightly different in next versions.

Do we adjust for that now, or do it after the next releases of 3.3 and 4.6 ?

Reply:
I used https://github.com/ibexa/elasticsearch/blob/v4.6.7/src/bundle/Resources/config/default-config.yaml#L73-L76 for the template so we're OK for this PR. I'll take time to check while adapting to 3.3.